core/src/Cabana_DeepCopy.hpp

```diff
@@ -141,6 +141,56 @@ namespace Experimental
 template<class Space, class SrcAoSoA>
 inline
 SrcAoSoA
+create_mirror_view(
+    const Space&,
+    const SrcAoSoA& src,
+    typename std::enable_if<(is_aosoa<SrcAoSoA>::value &&
```

Comment:
`is_aosoa<SrcAoSoA>::value` is superfluous.  I would drop it.  You may add a static assert if you like.

Reply:
fixed